dll/cpl/access/lang/pt-PT.rc

```diff
@@ -262,16 +262,16 @@ END
 
 IDD_SERIALKEYSOPTIONS DIALOGEX 0, 0, 246, 228
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
-CAPTION "Configurações de dispositivos seriais"
+CAPTION "Definições para ""Teclas série"""
 FONT 8, "MS Shell Dlg"
 BEGIN
-    GROUPBOX "Teclas seriais", -1, 6, 11, 234, 92
-    LTEXT "Escolha a porta para a ligação de um dispositivo de entrada alternativo.",
+    GROUPBOX "Teclas série", -1, 6, 11, 234, 92
+    LTEXT "Seleccione a porta para ligação de um dispositivo de entrada alternativo.",
           -1, 12, 20, 222, 20
-    LTEXT "&Porta serial:", -1, 12, 34, 222, 20
+    LTEXT "&Porta série:", -1, 12, 34, 222, 20
```

Comment:
https://pt.wikipedia.org/wiki/Interface_serial
ou "porta serial".

Reply:
Okay, well, serial is a word adopted from English brought by computer science. (in Portuguese "série")

Reply:
@HBelusca ready to aprove?